src/lib/components/FloatingArrow/FloatingArrow.test.svelte.ts

```diff
@@ -17,4 +18,58 @@ describe('FloatingArrow', () => {
 			expect(component).toBeInTheDocument();
 		}),
 	);
+
+	it(
+		'renders position based on context placement',
+		withEffect(() => {
+			const arrowRef = document.createElement('div');
+			const floating = useFloating({ placement: 'left' });
+			const { getByTestId } = render(FloatingArrow, {
+				props: { ref: arrowRef, context: floating.context, width: 20, height: 20 },
```

Comment:
You can pass props a little easier (it's valid, but more verbose). You can instead directly pass the component props:
```js
render(FloatingArrow, { ref: arrowRef, context: floating.context, width: 20, height: 20 });
```

Reply:
I tried this originally, but it was failing. So I opted for the nested `props` object as shown.

![Screenshot 2024-05-10 at 4 20 11 PM](https://github.com/skeletonlabs/floating-ui-svelte/assets/1509726/9fb92022-7320-4307-bd1d-ae3c5410535d)

Reply:
Ah that's an unlucky collision then, this works too